src/convert_bin2txt.py

```
@@ -0,0 +1,343 @@
+        except EndOfFile as err:
+            if (err.val > 0):
+                print("Input file is truncated:", path_to_syscalls_table_dat)
+            break
+        except:
+            print("Unexpected error:", sys.exc_info()[0])
+            raise
+        else:
+            sc_table.append(syscall)
+
+    fh.close()
+    return sc_table
+
+###############################################################################
+# read_log_entry - read next log entry
+###############################################################################
+
+def read_log_entry(fh, sized):
+
+    bdata = fh.read(sized)
+    if (len(bdata) < sized):
+        raise EndOfFile(len(bdata))
+    return bdata
+
+###############################################################################
+# get_n_paths -- get number of paths
+###############################################################################
+
+def get_n_paths(mask):
+
+    global Arg_path_mask
+
+    npaths = 0
+    for n in range(0, 6):
+        if (mask & Arg_path_mask[n] == Arg_path_mask[n]):
+            npaths += 1
+    return npaths
+
+###############################################################################
+# is_path -- checks if the argument is a path
+###############################################################################
+
+def is_path(n, mask):
+
+    global Arg_path_mask
+
+    if (mask & Arg_path_mask[n] == Arg_path_mask[n]):
+        return 1
+    else:
+        return 0
+
+###############################################################################
+# print_path -- print path
+###############################################################################
+
+def print_path(pathn, npaths, bdata):
+
+    if (npaths > 2):
+        raise NotImplementedError
+
+    pathn += 1
+
+    if (npaths == 2):
+        if (pathn == 1):
+            bdata = bdata[0:PATH1_MAX]
+        elif (pathn == 2):
+            bdata = bdata[PATH1_MAX:PATH2_MAX]
+
+    path = str(bdata.decode(errors="ignore"))
```

Are you sure about ignoring errors?